reference/conanfile.rst

```diff
@@ -504,6 +524,35 @@ That will produce a **conaninfo.txt** file like:
    of packages when upstream 0.Y.Z dependencies change, even for patches. Change it in your
    conan_info() method if you need to.
 
+Relative imports
+----------------
+If you want to reuse common functionality between different packages, it can be written in their
+own python files and imported from the main ``conanfile.py``. Lets write for example a ``msgs.py``
+file and put it besides the ``conanfile.py``:
+
+..  code-block:: python
+
+   def build_msg(output):
+      output.info("Building!")
+
+And then the main ``conanfile.py`` would be:
+
+..  code-block:: python
+
+   from conans import ConanFile
+   from msgs import build_msg
+
+   class ConanFileToolsTest(ConanFile):
+       name = "test"
+       version = "1.9"
+       exports = "msgs.py"  # Important to remember!
+   
+       def build_msg(self):
```

Comment:
I think its a better example to call build msg from build method (or other). Repeat method name is confusing